src/WebJobs.Extensions.DurableTask.Analyzers/Analyzers/Entity/DispatchClassNameAnalyzer.cs

```diff
@@ -40,17 +40,15 @@ private void AnalyzeDispatchEntityName(SyntaxNodeAnalysisContext context)
                 var name = expression.Name;
                 if (name.ToString().StartsWith("DispatchAsync"))
                 {
-                    if (TryGetTypeArgumentList(expression, out SyntaxNode typeArgumentList))
+                    if (SyntaxNodeUtils.TryGetTypeArgumentList(expression, out SyntaxNode identifierNode))
                     {
                         if (SyntaxNodeUtils.TryGetClassSymbol(expression, context.SemanticModel, out INamedTypeSymbol classSymbol))
                         {
                             var className = classSymbol.Name.ToString();
-
-                            //TypeArgumentList will always have a child node
```

Comment:
Was this not true?

Reply:
I moved that comment into a method I created in SyntaxNodeUtils called TryGetTypeArgumentList